src/integrationTest/java/uk/gov/hmcts/darts/testutils/stubs/UserAccountStub.java

```diff
@@ -28,7 +29,7 @@ public class UserAccountStub {
 
     private static final int SYSTEM_USER_ID = 0;
     public static final String INTEGRATION_TEST_USER_EMAIL = "integrationtest.user@example.com";
-    private static final String SEPARATE_TEST_USER_EMAIL = "separateintegrationtest.user@example.com";
+    public static final String SEPARATE_TEST_USER_EMAIL = "separateintegrationtest.user@example.com";
```

Comment:
Can this be `private`?

Reply:
it's now used in SecurityGroupControllerIntTest.java in the same way as INTEGRATION_TEST_USER_EMAIL so needs to be public